content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md

```diff
@@ -64,3 +64,26 @@ This will cause the following in the terminal:
 ![Terminal window showing the breakpoint() formula](https://raw.githubusercontent.com/Codecademy/docs/main/media/python-built-in-functions-breakpoint-screenshot.png)
 
 The execution of the program was paused, showing a prompt in the terminal. At this point, the code can be analyzed using the previously mentioned commands.
+
+## Codebyte Example
+
+Here's an example to demonstrate the usage of `breakpoint()`:
```

Comment:
@SSwiniarski @yangc95 @KTom101 This example throws an error when I run it on the docs compiler (it's a relatively new method, introduced in python 3.7). Should we omit this from the codebyte additions?

Reply:
I would suggest the same, also we need to understand pdb properly to use breakpoint to further debug the code. So after the breakpoint example we also have to give an example where we are using pdb. Like:
```python
import pdb
def calculate_sum(numbers):
    total = 0
    for number in numbers:
        total += number
        pdb.set_trace()  
    return total
```